Control.Geocoder.css

```diff
@@ -48,7 +53,7 @@
   height: 26px;
   border: none;
   background-color: white;
-  background-image: url(images/geocoder.png);
+  background-image: var(--magnifier);
```

Comment:
To not loose [IE11 support](https://caniuse.com/css-variables), please specify the `url` directly on `background-image`.

Reply:
I'd like to change that, but first I'd like to point out that microsoft's official support for IE11 will end this month. So in my opinion it is time to stop worrying about this browser. Why should we continue to support it when the manufacturer doesn't do it anymore.

Reply:
I know. In the past, we've got some bug reports of broken IE11 compatibility. In this case it does make a difference, and even saves a few lines. :)

Reply:
Okay, I'll change that 🙂

Reply:
Done.